dependencies/code_builder/buildspec.yml

```diff
@@ -0,0 +1,55 @@
+version: 0.2
+
+phases:
+  pre_build:
+    commands:
+      - echo "Logging in to the docker"
+  build:
+    commands:
+       - echo "Building and pushing Docker images"
```

Comment:
the message in the echo is different from the operations followed. Please use the proper comments

Reply:
It's not required, we ccan remove it